tests/fake_services/http_engine.py

```diff
@@ -82,10 +82,10 @@ def filter(self, data: str) -> Tuple[bytes, int]:
         mapped_fields = payload.get("mapped_fields")
         if payload.get("meeting_id"):
             raise ValueError("The argument meeting_id is not supported here.")
-        result: List[Dict[str, Any]] = []
-        if not filter.get("operator") == "==":
+        result: Dict[int, Dict[str, Any]] = {}
+        if not filter.get("operator") == "=":
```

Comment:
Can you define constants for these? The fact, that there was a mistake and several places needed adjustments ist a good opportunity for it ;)

Reply:
Wait, do you mean the operators? I think they are sufficiently verbose that no constants are needed. It's especcially nice that you can do `FilterOperator(field, "=", value)` which is about as verbose as it can get, while it would be less verbose to write `FilterOperator(field, Operator.EQUALS, value)` IMO

Reply:
Yes I mean exactly this. Otherwise there is room for mistakes. It is more verbose, but maybe you should do it this way and do not use `FilterOperator` directly: Make child classes (Is this a class?) like `EqualOperator`, and so on. Encapsulate the usage. Imagine we change `=` to `~` -> You hav to change may places in the code. With the constant it is easy. With the more abstration it is even better. E.g. special docs can be provided for each operator, how they behave. Like: What happens, if you compare a number and a string. It is specified in the DS interface, but it might be done here in the docs, too, so foreign developers can directly see how the `EqualOperator` behaves.

Reply:
I see where you are coming from, but I don't agree. We will most likely never touch these again, except to add more operators. If you still think it is that important, I can do it, but I don't see the necessity :D